apps/web/src/hooks/useContributors.ts

```
@@ -0,0 +1,112 @@
+			const monthNames = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"];
+
+			data.forEach(week => {
+				const weekDate = new Date(week.week * 1000);
+				const key = `${weekDate.getFullYear()}-${weekDate.getMonth()}`;
+				commitsByMonth[key] = (commitsByMonth[key] || 0) + week.total;
+			});
+
+			const allKeys = Object.keys(commitsByMonth)
+				.map(key => {
+					const [year, month] = key.split("-").map(Number);
+					return new Date(year!, month!, 1);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)